### ISSUE TITLE
Coordinate system consistency

### ISSUE THREAD

**Issue by ctchervenkov** [matsim-eth/baseline_scenario#37]
Different source data used to create the scenario express coordinates in different reference frames (sometimes CH1903, sometimes WGS84, etc.).
Should we convert all this raw data such everything is expressed in CH1903+ from the beginning, or should we trust that we make the proper conversion later?

**Comment by tduberne** [comment]
Yes, we should have it all in the same projection. The reason is that the CRS can be converted, but one has to specify it in the config file, not the file itself, which opens the way to forgetting/copy-paste error.

**Comment by sebhoerl** [comment]
I think the queston is more abou the raw data, such as the microcensus. My clear answer there is no, because the idea is to simply have these files as we get them (hence "raw") and then use them. Since we know the projections which are used in the data sets, this is fine. However, for each of those data sets, when they are used one of the first steps should be to convert them to a common projection, for which we should always use `CH1903+ / LV95`.

**Comment by tduberne** [comment]
Oh, I see... I agree, the "raw" data should remain exactly as obtained, and the processing pipeline should take care of re-projecting everything in the same system as a first step.
